go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -562,9 +562,18 @@ func (npw *nodePortWatcher) generateARPBypassFlow(ofPorts []string, ofPortPatch,
 			}
 			arpPortsFiltered = append(arpPortsFiltered, port)
 		}
-		arpFlow = fmt.Sprintf("cookie=%s, priority=110, in_port=%s, %s, %s=%s, "+
-			"actions=output:%s",
-			cookie, npw.ofportPhys, addrResProto, addrResDst, ipAddr, strings.Join(arpPortsFiltered, ","))
+
+		// If vlan tagged traffic is received from physical interface, it has to be untagged before sending to access ports
+		if config.Gateway.VLANID != 0 {
+			match_vlan := fmt.Sprintf("dl_vlan=%d,", config.Gateway.VLANID)
+			arpFlow = fmt.Sprintf("cookie=%s, priority=110, in_port=%s, %s,%s, %s=%s, "+
```

Comment:
super NIT: add a space after the new `%s, ` or better yet, spell out the field to be consistent with in_port:
```
arpFlow = fmt.Sprintf("cookie=%s, priority=110, in_port=%s, dl_vlan=%d, %s, %s=%s, "+
"actions=strip_vlan,output:%s",
				cookie, npw.ofportPhys, config.Gateway.VLANID, addrResProto, addrResDst, ipAddr, strings.Join(arpPortsFiltered, ","))
```

Reply:
Fixed by adding space